### PR TITLE
Publish Package 5.2.0

### PR DESCRIPTION
**Pull request by ektaghag-eaton**
Publish Package 5.2.0

#### Changes proposed in this Pull Request:

- This PR contains changes for 6.0.0-alpha.1 (Published By the Fiji team)
- Package 5.2.0 (Fixes for Fiji team)
-

#### Screenshots / Screen Recording (if applicable)

-

#### To Test:

- **For Version 6.0.0-alpha.1**

-   Ability to hide the CreatePassword screen.
-   Ability to hide the login form in the Login screen.

- For Version 5.2.0

- Disable Next button when valid prop of Account Details form is false ([#276](https://github.com/etn-ccis/blui-react-native-workflows/issues/276)).
-   Verification code prefilled value issue on Verify Email screen on resend, next and back button pressed ([#280](https://github.com/etn-ccis/blui-react-native-workflows/issues/280)).

-   Incorrect navigation on dismissal of error occurred in Account Details screen ([#275](https://github.com/etn-ccis/blui-react-native-workflows/issues/275)).

#### Any specific feedback you are looking for?

- In **5.2.0 version** changes from **6.0.0-alpha.1** will be available . @joebochill is it fine?

### CODE REVIEW

File: login-workflow/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,22 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v6.0.0 (Unreleased)
```
Comment (JeffGreiner-eaton) [inline login-workflow/CHANGELOG.md:8]: should this be updated to be v6 with date or is this for alpha?

Comment (ektaghag-eaton) [reply login-workflow/CHANGELOG.md:8]: V6 is still in the alpha version, we haven’t published 6.0.0.

Comment (joebochill) [reply login-workflow/CHANGELOG.md:8]: v6 should be removed from the changelog...those changes that were part of the original 6.0.0-alpha were incorporated into 5.2.0. So there will be no unreleased changes after 5.2.0 is published.

File: login-workflow/package.json [hunk 2]
```diff
@@ -57,7 +57,7 @@
         "tag:package": "npx -p @brightlayer-ui/tag blui-tag -s -blui-react-native-auth-workflow"
     },
     "dependencies": {
-        "@brightlayer-ui/react-auth-shared": "^3.7.3",
+        "@brightlayer-ui/react-auth-shared": "^4.0.0-alpha.2",
```
Comment (joebochill) [inline login-workflow/package.json:60]: We cannot use pre-release versions of dependencies in a latest version of our packages.

**Review by joebochill — CHANGES_REQUESTED** [review]

**Comment by JeffGreiner-eaton** [comment]
been trying to run start:example-ios here and getting this error on fresh clone. My local issue?

image

**Comment by joebochill** [comment]
@JeffGreiner-eaton looks like maybe a local cocoapods version issue. I'm able to clone and run (once i disable hermes in the podfile).

**Review by joebochill — APPROVED** [review]

**Review by JeffGreiner-eaton — APPROVED** [review]
approving as secondary on review to get merged

File: login-workflow/package.json [hunk 3]
```diff
@@ -57,7 +57,7 @@
         "tag:package": "npx -p @brightlayer-ui/tag blui-tag -s -blui-react-native-auth-workflow"
     },
     "dependencies": {
```
Comment (joebochill) [inline login-workflow/package.json:59]: We should avoid making dependency changes in a minor version if we can. Android currently fails to run (seems to be related to the updated version of react-native-gesture-handler).

**Review by joebochill — CHANGES_REQUESTED** [review]

**Review by joebochill — APPROVED** [review]

**Review by surajeaton — APPROVED** [review]

**Pull request merged**